<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statistics</class>
 <widget class="QDialog" name="statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>统计界面</string>
  </property>
  <property name="styleSheet">
   <string notr="true">statistics{background-image: url(:/new/myimage/new_icon/Background.jpg);}</string>
  </property>
  <widget class="QPushButton" name="save_date">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="start_date">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="end_date">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>461</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(187, 203, 216,200);
font: 16pt &quot;华文楷体&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>收入</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>441</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="income_layout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="no_income">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>支出</string>
    </attribute>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>441</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="outcome_layout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="no_outcome">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="go_back">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_choose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>请选择查询时间段：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_start_date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>31</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>起始日期：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_end_date">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>STKaiti</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结束日期：</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
